### ISSUE TITLE
error parsing mdstatus: error parsing mdstat: too few matches found in status

### ISSUE THREAD

**Issue by amoskong** [scylladb/scylla#2097]
*Installation details*
Scylla version (or git commit hash): 1.7.rc0-0.20170207.0429e5d  ( I also met this error in 1.6 testing)
Cluster size: 6
OS (RHEL/CentOS/Ubuntu/AWS AMI): CentOS

I found many error from node_exporter in longevity test of 1.6 & 1.7
Question: Is it harmful?   @avikivity  @@syuu1228

Error detail:
```
db node # journalctl
...
Feb 21 02:42:08 longevity-1-7-gce-scylla-db-node-d9ea4a62-003 node_exporter[417]: time="2017-02-21T02:42:08Z" level=error msg="ERROR: mdadm collector failed after 0.000092s: error parsing mdstatus: error parsing mdstat: too few matches found in statusline:       393084928 blocks super 1.2 1024k chunks" source="node_exporter.go:91"
```

**Comment by syuu1228** [comment]
Not sure detail of node_exporter, but it seems trying to parse /proc/mdstat on their code:
https://github.com/prometheus/node_exporter/blob/master/collector/mdadm_linux.go#L135

Perhaps corrupted parser or format is different between kernels?

**Comment by slivne** [comment]
Lets open a bug on node_exporter upstream or send this on the user mailing
list

Weare pushing users to install this / installing this by default on the ami
we need this resolved

**Comment by slivne** [comment]
prometheus/node_exporter#500

**Comment by slivne** [comment]
and we have an answer

> Please update your node_exporter version, yours is quite old. I'm pretty sure raid0 parsing has been fixed in 0.13.0

**Comment by slivne** [comment]
@amnonh / @syuu1228 - can we upgrade to the newver version ours is 0.12 and test
